substrate/client/sysinfo/src/lib.rs

```diff
@@ -133,6 +138,7 @@ pub fn print_sysinfo(sysinfo: &sc_telemetry::SysInfo) {
 /// Prints out the results of the hardware benchmarks in the logs.
 pub fn print_hwbench(hwbench: &HwBench) {
 	log::info!("🏁 CPU score: {}", hwbench.cpu_hashrate_score);
+	log::info!("🏁 CPU parallelism score: {}", hwbench.parallel_cpu_hashrate_score);
```

Comment:
Maybe you can merge the two lines so that it prints single and multi thread performance in one line?